plugin/storage/integration/kafka_test.go

```diff
@@ -99,6 +99,9 @@ func (s *KafkaIntegrationTestSuite) initialize() error {
 	}
 	spanConsumer.Start()
 
+	//wait consumer running
+	time.Sleep(time.Second * 5)
```

Comment:
we don't do this in tests. If we need to wait for a specific condition, then we can use assert.Eventually(), which will exit as soon as condition is met. Unconditionally waiting for 5 seconds makes the tests slow for now reason.

Reply:
use spanConsumer.Ready(),  The new ready function indicates that the consumer is ready